src/SWP/Component/MultiTenancy/Model/AbstractOrganization.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
And it should also be possible to have an organization without a tenant? This code is flexible, but unless all permutations are actually supported, you're really just making the code unnecessarily complex, I think.